src/scripts/OSFramework/OSUI/ErrorCodes.ts

```diff
@@ -18,12 +18,11 @@ namespace OSFramework.OSUI.ErrorCodes {
 	export const Dropdown = {
 		FailOptionItemClicked: 'OSUI-GEN-02001',
 		FailOptionItemKeyPressed: 'OSUI-GEN-02002',
-		FailRegisterCallback: 'OSUI-GEN-02003',
-		FailSetNewOptionItem: 'OSUI-GEN-02004',
-		FailToSetOptionItemAction: 'OSUI-GEN-02005',
-		FailUnsetNewOptionItem: 'OSUI-GEN-02006',
+		FailSetNewOptionItem: 'OSUI-GEN-02003',
+		FailToSetOptionItemAction: 'OSUI-GEN-02004',
+		FailUnsetNewOptionItem: 'OSUI-GEN-02005',
 		HasNoImplementation: {
-			code: 'OSUI-GEN-02007',
+			code: 'OSUI-GEN-02006',
```

Comment:
In my opinion this is not a good decision, this codes shouldn't be changed or reorder. In my opinion the name can change a bit but not the context. the code 02003 could be just commented and leave the rest of the codes has they were.

Reply:
This is to facilitate the debug and analysis between different versions